libs/sdk-core/src/persist/reverseswap.rs

```diff
@@ -25,30 +25,39 @@ impl SqliteStorage {
         )?;
 
         tx.execute(
-            "INSERT INTO reverse_swaps_info (id, status)\
-            VALUES (:id, :status)",
+            "INSERT INTO reverse_swaps_info (id, status, lockup_txid, claim_txid)\
+            VALUES (:id, :status, :lockup_txid, :claim_txid)",
             named_params! {
                 ":id": rsi.id,
-                ":status": serde_json::to_value(rsi.cache.status)?
+                ":status": serde_json::to_value(rsi.cache.status)?,
+                ":lockup_txid": "",
+                ":claim_txid": "",
             },
         )?;
 
         tx.commit()?;
         Ok(())
     }
 
-    pub(crate) fn update_reverse_swap_status(
+    pub(crate) fn update_reverse_swap_cache(
         &self,
         id: &str,
         status: &ReverseSwapStatus,
+        lockup_txid: &str,
+        claim_txid: &str,
```

Comment:
These could be optional strings instead of passing empty strings.
```suggestion
        lockup_txid: Option<String>,
        claim_txid:  Option<String>,
```

Reply:
No longer applicable after addressing another comment.